WordPress/wordpress-playground
# [Website] Nightly build bugfix – ship the actual nightly build, not the latest release

Fixes a regression created in #2077. The minified nightly build accidentally became a minified build of the latest WordPress version.

In the multitude of resolveWordPressRelease() calls used in building the minified WordPress releases, the last call in the chain started transforming the nightly URL into a 6.7.1 release URL. This PR removes that one final call, and also simplifies the Playground CLI build pipeline.

 ## Testing instructions

Apply this PR, switch WP version to nightly, confirm in wp-=admin it's indeed the nightly release
